src/span_context.js

```diff
@@ -82,6 +89,14 @@ export default class SpanContext {
         this._baggage = baggage;
     }
 
+    set debugId(debugId: string): void {
+        this._debugId = debugId;
+    }
+
+    isDebugIDContainerOnly(): boolean {
+        return !this._traceId && this._debugId !== '';
```

Comment:
I use ! because when i initialize an empty span context, all of its values will be undefined.  So traceId will always be undefined.